source/lib/src/prod_env_mat.cc

```diff
@@ -320,3 +320,68 @@ void deepmd::env_mat_nbor_update(
   delete [] mesh_host;
 }
 #endif // GOOGLE_CUDA
+
+#if TENSORFLOW_USE_ROCM
+void deepmd::env_mat_nbor_update(
+    InputNlist &inlist,
+    InputNlist &gpu_inlist,
+    int &max_nbor_size,
+    int* &nbor_list_dev,
+    const int * mesh, 
+    const int size)
```

Comment:
Can this function merged with the cuda version?  I find it should be possible.

Reply:
because merge this function with cuda, thus we will get the structure：
#if GOOGLE_CUDA
.......
#endif //GOOGLE_CUDA

#IF TENSORFLOW_USE_ROCM
........
#ENDIF//TENSORFLOW_USE_ROCM

#if TENSORFLOW_USE_ROCM || GOOGLE_USE_ROCM
this function
# endif //
I didn't use this structure when I migrated without wanting to change the code that I had written before